app/scripts/controllers/overview.js

```diff
@@ -1349,7 +1349,9 @@ function OverviewController($scope,
       updateQuotaWarnings();
     }, {poll: true, pollInterval: DEFAULT_POLL_INTERVAL}));
 
-    if (SERVICE_CATALOG_ENABLED) {
+    var canI = $filter('canI');
+    // The canI check on watch should be temporary until we have a different solution for handling secret parameters
```

Comment:
We want instances to be watchable by project viewers, once they are then this check isn't really necessary, we dont have this check for any of the other watches today.  We can't make them watchable (or listable) by project viewers yet because of the way parameters are are currently stored on instances and bindings.  Viewers today should not be allowed to see potentially confidential information.